website/templates/profile/notifications.mako

```diff
@@ -54,6 +54,10 @@
         </div>
         <div class="panel panel-default">
             <div class="panel-heading clearfix"><h3 class="panel-title">Configure Notification Preferences</h3></div>
+            <div class="panel-body">
+                <div class="help-block" style="padding-left: 15px">
```

Comment:
Minor: With `panel-body` added, the extra padding should no longer be necessary. I can remove this when I merge.